### PR TITLE
Portfolio Project: Works and About me section

### PR DESCRIPTION
**Pull request by Devtiwo**
In this project, I :

- Added works and about me sections to the index.html file
- Added styles to the new sections that were created in the index.html file
- Added more images to the images folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,7 +25,123 @@ <h3>I am a software Developer</h3>
              <li><a href="https://www.linkedin.com/in/taiwo-adetunji-860666225"><img src="images/linkedin.png" class="icons" alt="linkedin logo"></a></li>
              <li><a href="https://twitter.com/devtiwo"><img src="images/Vector.png" class="icons" alt="twitter logo"></a></li>
          </ul>
-        </section>
+     </section>
+     <section class="works-header">
```
Comment (elyor-doniyorov) [inline index.html:29]: - [ ] Kindly put all the code related to your works into one main `<section>` tag(from the line `29` to `106`). You now have them separated from each other.

File: index.html [hunk 2]
```diff
@@ -25,7 +25,123 @@ <h3>I am a software Developer</h3>
...
+         <article class="frame">
+             <h2>Profesional Art Printing Data</h2>
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <div class="wrapper">
+                 <div class="items">html</div>
+                 <div class="items">bootstrap</div>
+                 <div class="items">Ruby</div>
+             </div>
+             <button class="btn">See Project</button>
+         </article>
```
Comment (elyor-doniyorov) [inline index.html:55]: - [ ] Make sure for each project:
* The content of the project is aligned with flexbox
* Button styling: Active, Hover, and Pressed (see design)
* The list of project technologies (e.g., "HTML", "Bootstrap" and "Ruby") are a list `ul` and each of them a `li`

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Devtiwo, 

Good job so far!

## To highlight 👏 

- Descriptive Pull Request title and description ✅ 
- There are no linter errors ✅ 
- Correct use of GitHub flow ✅ 

There are some issues that you still need to work on to go to the next project but you are almost there!
keep-trying-youre-almost-there


## Required Changes ♻️

- [ ] Your app has a horizontal scroll. Kindly fix it 👇 
Screenshot (36)_LI

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Capitalize the first letter of your upcoming commits;
- Add valid live demo and issues page links

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -25,7 +25,125 @@ <h3>I am a software Developer</h3>
...
+                     <li class="items">bootstrap</li>
+                     <li class="items">Ruby</li>
+                 </ul>
+                 <button class="btn">See Project</button>
+             </article>
+         </section>
+      </section>
+      <section class="about">
+          <article class="myself">
+             <h1>About Me</h1>
```
Comment (rotimiazeez) [inline index.html:111]: - [OPTIONAL] As stated earlier kindly make sure you have only one `h1` on the page you can use other `hx` tag `h2-h6` 👌👌

**Review by rotimiazeez — COMMENTED** [review]
# Hi <STUDENT_MENTION>, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Kindly make the optional changes before your next rewiew😉😉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi <STUDENT_MENTION>, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Kindly make the optional changes before your next rewiew😉😉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**